vignettes/pillar.Rmd

```diff
@@ -0,0 +1,458 @@
+---
+title: "Printing vectors nicely in tibbles"
+author: "Kirill Müller, Hadley Wickham"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Printing vectors nicely in tibbles}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+options(crayon.enabled = TRUE)
+options(pillar.bold = TRUE)
+
+knitr::opts_chunk$set(collapse = TRUE, comment = pillar::style_subtle("#>"))
+
+colourise_chunk <- function(type) {
```

Comment:
I'd prefer to leave this off to match the other vignettes (and you appear to have already included a caveat about colouring not working)